addons/project/views/project_sharing_views.xml

```diff
@@ -206,8 +198,7 @@
                                     <field name="display_project_id" string="Project" optional="hide" invisible="1"/>
                                     <field name="company_id" invisible="1"/>
                                     <field name="partner_id" options="{'no_open': True, 'no_create': True, 'no_edit': True}" optional="hide"/>
-                                    <!-- TODO: [XBO] remove me in master -->
-                                    <field name="user_ids" invisible="1"/>
+                                    <field name="user_ids" invisible="1" />
```

Comment:
I didn't remove it because we need it to correctly set the portal user by default.